.github/workflows/test.yml

```
@@ -0,0 +1,99 @@
+name: Run tests
+on:
+  workflow_dispatch:
+  push:
+    branches:
+    - master
```

```suggestion
  push:
    paths-ignore:
  	- 'html/changelogs/**'
    branches:
    - master
```

We don't want it to run for CL pushes